script.js

```diff
@@ -0,0 +1,91 @@
+let itemList = [];
+
+// 로컬 스토리지에서 값을 불러와 화면에 그려 준다
+const renderTodoItem = () => {
+  const savedTodo = localStorage.getItem("itemList");
+
+  const todoList = document.getElementById("todo-list");
+  const doneList = document.getElementById("done-list");
+
+  // 덮어쓰지 않도록 초기화
+  todoList.innerHTML = "";
+  doneList.innerHTML = "";
+
+  // 데이터가 존재하는지 확인
+  if (savedTodo) {
+    itemList = JSON.parse(savedTodo);
+
+    itemList.forEach((savedItem) => {
+      const item = document.createElement("li");
+
+      const itemText = document.createElement("span");
+      itemText.className = "item-text";
+      itemText.addEventListener("click", toggleItem);
+      itemText.innerText = savedItem.text;
+
+      const deleteButton = document.createElement("button");
+      deleteButton.className = "delete-button";
+      deleteButton.addEventListener("click", removeItem);
+      deleteButton.innerText = "🧹";
+
+      item.appendChild(itemText);
+      item.appendChild(deleteButton);
+
+      if (!savedItem.isDone) {
+        todoList.appendChild(item);
+        // deleteButton.appendChild(todoList);
+      } else {
+        doneList.appendChild(item);
+        //deleteButton.appendChild(item);
+      }
+    });
+  }
+  countItem();
+};
+
+// 새로운 할 일을 입력 받을 때 로컬 스토리지에 추가한다
+const addTodoList = () => {
+  event.preventDefault;
```

Comment:
`addTodoList`에서 event.preventDefault() 처리가 안되고 계속 새로고침이 일어나는 것 같아요! event를 인자로 받아주시고 호출 인자에 이벤트를 부여해주셔야 할 것 같아요!
```suggestion
const addTodoList = (event) => {
  event.preventDefault();
```

Reply:
저도 이거 제출하고 나서야 깨달았네요... 수정하겠습니다!!